src/services/school/services/schoolsList.js

```diff
@@ -12,7 +12,8 @@ class SchoolsListService {
 		};
 		const systemsQuery = {
 			path: 'systems',
-			select: '_id type alias',
+			select:
+				'_id type alias oauthconfig.client_id oauthconfig.auth_endpoint oauthconfig.code_redirect_uri oauthconfig.response_type oauthconfig.scope',
```

Comment:
add an enabled flag to oauthconfig which controls the visibility of the button in the client and if oauth config data is added here. we might need to set data while not enable the feature immediately. this must be checked when login, to be able to disable login while keeping configuration too.

Reply:
I would say, we can do that in the next release, since it is not critical now for the first release, due to the time pressure.